libaleth-precompiles/PrecompilesVM.h

```diff
@@ -0,0 +1,17 @@
+// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2018-2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
+#pragma once
+
+#include <evmc/evmc.h>
+#include <evmc/utils.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+EVMC_EXPORT struct evmc_vm* evmc_create_aleth_precompiles_vm() EVMC_NOEXCEPT;
```

Comment:
If the library name is `aleth-precompiles` this should be named `evmc_create_aleth_precompiles` for EVMC loader to work.